packages/twenty-ui/src/display/chip/components/Chip.tsx

```diff
@@ -34,9 +33,86 @@ type ChipProps = {
   rightComponent?: ReactNode;
   className?: string;
   onClick?: (event: MouseEvent<HTMLDivElement>) => void;
-  to?: string;
 };
 
+const StyledContainer = withTheme(styled.div<
```

Comment:
is this the recommended way to use theme? I can see THEME_COMMON import in other components

Reply:
For the moment yes because here we have the light/dark mode involved. But I'm working on a common way to import light/dark theme variables from context.